pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -468,6 +467,7 @@ func (s *Server) createIstioCA(opts *caOptions) (*ca.IstioCA, error) {
 
 		s.initCACertsWatcher()
 	}
+	caOpts.OnRootCertUpdate = s.updatePluggedinRootCertAndGenKeyCert
```

Comment:
The naming of `updatePluggedinRootCertAndGenKeyCert` also seems a bit confusing. The naming makes me think it shouldn't be used when the CA is self-signed and only when it is plugged in by the user. Could it instead be something like `updateDNSCertAndBundle`?

Reply:
Should we add a debug log line to `updatePluggedinRootCertAndGenKeyCert`?

Reply:
Yes, will do a little bit improve on this